criu/seize.c

```diff
@@ -254,6 +254,12 @@ static int seize_cgroup_tree(char *root_path, enum freezer_state state)
 			return -1;
 		}
 
+		if (pid_is_stopped(pid)) {
+			pr_err("unexpected state of %d: stopped\n", pid);
+			fclose(f);
+			return -1;
```

Comment:
Note that the THAWED writing can have non-immediate effect and some processes may still be stopped. Please see how we can retry here https://src.openvz.org/projects/OVZ/repos/criu/browse/criu/seize.c#510 or just wait for it to actually change like this https://src.openvz.org/projects/OVZ/repos/criu/browse/criu/seize.c#310 .

Reply:
Hi @Snorch Thanks for your reply.
We care about process in state ‘T’ or not. I found freezing process did not change 'T'  State.  

 ```
[root@my tmp]# mkdir  -p /sys/fs/cgroup/freezer/test
[root@my tmp]# sleep 1000 & 
[2] 12828
[root@my tmp]# kill -s SIGSTOP 12828

[2]+  已停止               sleep 1000
[root@my tmp]# cat /proc/12828/status  |grep State
State:  T (stopped)
[root@my tmp]# echo 12828 > /sys/fs/cgroup/freezer/test/tasks 
[root@my tmp]# echo FROZEN > /sys/fs/cgroup/freezer/test/freezer.state 
[root@my tmp]# cat /proc/12828/status  |grep State
State:  T (stopped)
```